BlazarService/pom.xml

```diff
@@ -249,7 +257,8 @@
             <configuration>
               <resources>
                 <resource>
-                  <directory>${project.build.directory}/generated-resources</directory>
+                  <directory>${project.basedir}/../BlazarUI/dist</directory>
```

Comment:
FYI gulp was giving me grief with it not wanting to clean folders outside of it's base path, i tweaked it to build everything in `dist` and then the `add-resource` goal here copies it into BlazarService